packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -118,6 +121,7 @@ export class MonacoEditorProvider {
 
         const options = this.createMonacoDiffEditorOptions(originalModel, modifiedModel);
         const editor = new MonacoDiffEditor(uri, document.createElement('div'), originalModel, modifiedModel, this.m2p, this.p2m, this.decorationsService, options, override);
+        this.doInitialDiffNavigation(editor);
```

Comment:
Is it necessary or it can be first initialized on demand?

Reply:
this implicitly reveals first diff, cf. `alwaysRevealFirst`.

OTOH if we're sure that other parts, like diff widget, will create a diff navigator for this diff editor, this is even redundant. but will this happen?

Reply:
it is fine, just asking